src/mail/export/Exporter.ts

```diff
@@ -67,7 +67,7 @@ export async function exportMails(
 	fileController: FileController,
 	operationId?: OperationId,
 	signal?: AbortSignal,
-): Promise<Mail[]> {
+): Promise<{ failures: Mail[] }> {
```

Comment:
sorry for nitpick but `failures` sounds like those would be errors, I would maybe say `failed`?